index.Rmd

```diff
@@ -249,12 +251,12 @@ as(MNDWI, "SpatialPixelsDataFrame") %>%
         axis.ticks = element_blank(),
         panel.background = element_blank(),
         panel.grid.minor = element_blank()) +
-  labs(title = "Water or Water ice or Volatiles materials", 
+  labs(title = "Water or Water Ice or Volatiles Materials", 
        x = " ", 
        y = " ") +
   scale_fill_gradient(high = "#CEE50E", 
-                      low = "#087F28",
-                      name = "Water or Water ice or Volatiles materials")
+                      low = "#087F28", 
```

Comment:
Same suggestion as my last suggestion where the colors should be flip flopped in order. 0 is usually the lightest value you see in a figure.